application/Modules/Products/Controllers/AjaxController.php

```
@@ -60,11 +60,16 @@ public function modal_product_lookups()
             'default_item_tax_rate' => $default_item_tax_rate,
         ];
 
+        // Capture view output and return as JSON
+        ob_start();
         if ($filter_product || $filter_family || $reset_table) {
             $this->layout->load_view('products/partial_product_table_modal', $data);
         } else {
             $this->layout->load_view('products/modal_product_lookups', $data);
         }
```

PHPMD (2.15.0)

The method modal_product_lookups uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)